src/Kernel32.Desktop/Kernel32Extensions.cs

```diff
@@ -14,16 +14,17 @@ public static partial class Kernel32Extensions
         /// Gets the text associated with an <see cref="NTSTATUS"/>.
         /// </summary>
         /// <param name="status">The error code.</param>
+        /// <param name="dwLanguageId">The language indentifier for the message</param>
         /// <returns>The error message. Or <c>null</c> if no message could be found.</returns>
-        public static string GetMessage(this NTSTATUS status)
+        public static string GetMessage(this NTSTATUS status, int dwLanguageId = 0)
```

Comment:
Default parameters are just C# syntax sugar. The metadata for the method stores default values as attributes. But the signature of the method itself is significantly changed by adding another parameter. The only thing adding a default parameter does that doesn't break compatibility is doesn't break *source* compatibility (that is, it wouldn't cause a compile error). But an application that already compiled against the original method would start throwing `MethodNotFoundException` because the original method is gone.
So if you're adding a brand new method, a default parameter is fine. But since you can't change existing method signatures without breaking binary compatibility, if you want to add a parameter you must add a new overload, and as such, it can't have a default value lest the C# compiler interpret it as ambiguous when you call the shorter signature one.